src/main/java/com/uber/cadence/samples/hello/HelloSearchAttributes.java

```diff
@@ -70,9 +68,54 @@
 
     @Override
     public String getGreeting(String name) {
+      SearchAttributes currentSearchAttributes = Workflow.getWorkflowInfo().getSearchAttributes();
+      System.out.println("Search Attributes on start: ");
```

Comment:
Use Workflow.getLogger instead.

Reply:
Logger output is not very clear for demo purpose (which will also contains some worker info) So I added comments to remind user to user logger in production.

Reply:
I would remove this line